docs/dev/fundamentals/rollups.md

```diff
@@ -0,0 +1,37 @@
+# Introduction
+
+To better understand rollups we need to dive briefly into Ethereum and Layer 2 solutions.
+
+The enormous demands from users to transact Ethereum put a limit on the network capacity by causing congestion, resulting in slow transactions and increased gas prices. While this has remained so for a while, an improved solution is needed, one that can maintain scalability.
+That is, to increase finality and achieve high transaction rates per second without having to trade off security. That's where layer 2 solutions shine.
+
+Layer 2 solutions are designed as an extension to Ethereum, offer various solutions poised to be the critical scalability solution to the inherent network congestion on Ethereum. Covering all Layer 2 solutions is beyond the scope of this doc.
+We will go through a brief explainer on rollups in this section.
+
+## Optimistic rollups versus ZK-rollups
+
+There are mainly two types of rollups, ZK-rollups and optimism rollups.
+
+### What are ZK rollups?
+
+ZK rollups ('ZK' standing for zero-knowledge) are a recent development intended to increase the scalability of Ethereum by performing calculations off-chain, rolling many transactions up into a single batch, and sending it to the main Ethereum chain for verification in a single action. 
```

Comment:
until the very end of the sentence, it just describes a rollup. I suggest to put this above, and here explain that the single batch is being verified by a cryptographic proof